contracts/toCKB-lockscript/src/main.rs

```diff
@@ -57,16 +51,41 @@ impl From<SysError> for Error {
 }
 
 fn main() -> Result<(), Error> {
-    // remove below examples and write your code here
+    verify()
+}
 
-    let script = load_script()?;
-    let args: Bytes = script.args().unpack();
-    debug!("script args is {:?}", args);
+fn verify() -> Result<(), Error> {
+    let script_hash = load_script_hash()?;
+    let args: Bytes = load_script()?.args().unpack();
 
-    let tx_hash = load_tx_hash()?;
-    debug!("tx hash is {:?}", tx_hash);
+    verify_toCKB_cells(Source::Input, &script_hash, args.as_ref())?;
+    verify_toCKB_cells(Source::Output, &script_hash, args.as_ref())?;
```

Comment:
try another way: use Source::Group to find cells, and verify all cell_typescript_hash[0..54] == args

Reply:
done, just use the GroupInput